packages/ia-prototype-apps/package.json

```diff
@@ -7,6 +7,7 @@
     "@internetarchive/ia-components": "^0.0.0-alpha.20+5039f8d",
     "@internetarchive/ia-js-client": "^0.0.0-alpha.20+5039f8d",
     "@internetarchive/scrubber-bar": "file:../scrubber-bar",
+    "@internetarchive/waveform-progress": "file:../waveform-progress",
```

Comment:
Question: why is this not pointing to the tag?

Reply:
It seems like this is how you import local repos for the demo app since I haven't pushed these  packages to NPM yet.

Reply:
I updated it to point to the tag.